fix: Unnecessary Spaces And Grammar

krishdevdb opened the pull request:
closes #1582 

#### What does this PR do?

resolves #1582 

#### Description of Task to be completed?

fix #1582 

#### How can this be manually tested?

See the diff:

```diff
diff --git a/youtube/generic.md b/youtube/generic.md
index 71ecb86..ee69d62 100644
--- a/youtube/generic.md
+++ b/youtube/generic.md
@@ -1,49 +1,57 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
 
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!
 
-## LEARN MORE
+## YouTube Shorts
+I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1
 
-I now have a 2nd YouTube channel for short video clips of 60s or less, subscribe ... 
-https://www.youtube.com/eddiejaoudetv
+## Newsletter
 
-JOIN our inclusive open source community EddieHub now for FREE ... http://eddiehub.org, also checkout my website and sign up to our community NEWSLETTER... http://eddiejaoude.io for my live stream + video schedule and community calls
+Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls
 
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
+Join our inclusive open source community EddieHub now for FREE at https://www.eddiehub.org
+ 
+\1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+\2. Choose The Issue Type Of Invite Me To The Organization
+\3. Choose The Issue Type Of Invite Me To The Organization
+\4. Open The Issue
+\5. You get an invitation to the community by our GitHub Action 🎉
 
-## SERVICES I USE AND RECOMMEND
+---
 
-Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
-
-- Live streaming with Streamyard http://links.eddiejaoude.io/t/pYlzGjcta1a4
-- Cloud hosting with Digital Ocean, you will get $100 FREE credit http://links.eddiejaoude.io/t/EAaCRWhsnZlF
-- NoSQL serverless database with DataStax, you will get $25 FREE credit every month http://links.eddiejaoude.io/t/bBM0STvCi2a4
-- Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
-
-## COMMUNITY SPONSORS
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
 Stephen Mount http://github.com/stemount
 Nicholas Carrigan https://github.com/nhcarrigan
-***
+Santosh Yadav https://github.com/santoshyadavdev
+Andrew Cunliffe https://github.com/andrew-cunliffe
+Alexander Stoichkov https://github.com/SashoStoichkov
+Allan Regush https://github.com/AllanRegush
 
-- Santosh Yadav https://github.com/santoshyadavdev
-- Andrew Cunliffe https://github.com/andrew-cunliffe
-- Alexander Stoichkov https://github.com/SashoStoichkov
-- Allan Regush https://github.com/AllanRegush
+---
 
-## FIND ME ON OTHER SOCIAL PLATFORMS
+## Service I Use and Recommend
 
-Don’t forget to support my channel by subscribing below, it’s free, and also share with your friends. Subscribe now!!
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
 
-http://eddiejaoude.io 
-https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+Live streaming with Streamyard: http://links.eddiejaoude.io/t/pYlzGjcta1a4
+Cloud hosting with Digital Ocean, you will get $100 FREE credit: http://links.eddiejaoude.io/t/EAaCRWhsnZlF
+NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
+Membership, Newsletters, CRM all in one with Kartra http://links.eddiejaoude.io/t/pQ5W3FRLk8Cp
+
+---
+ 
+## Find me on social media
+
+Website: https://www.eddiejaoude.io 
+GitHub: https://github.com/eddiejaoude
+YouTube: https://youtube.com/eddiejaoude
+Twitter: https://twitter.com/eddiejaoude
+Instagram: https://instagram.com/eddiejaoude
+LinkedIn: https://linkedin.com/in/eddiejaoude
+GitHub Community Org: https://github.com/EddieHubCommunity
+
+## Further
+Join Our Conversation At Discord: http://discord.eddiehub.org/
```

#### Any background context you want to provide?

Duplicate of #1615 made due to some issue see in the pull request comments

#### What is the relevant issue link?

#1582

#### Screenshots (if appropriate)

<img src="https://user-images.githubusercontent.com/30869493/117687590-3bc81800-b1b8-11eb-8b94-6b539aef0d42.jpg" width="300">

krishdevdb commented:
@Vyvy-vi the changes you asked in the previous pull request are made in this one.

Tagging @schmelto cause he opened the issue and I think his views matter the most!

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: [youtube.com/eddiejaoude](https://youtube.com/eddiejaoude?sub_confirmation=1)
```
afaik Markdown does **not** work in youtube description.
The entire point of this file is so that Eddie can copy-and-paste the text onto the description section.
```suggestion
Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```
It would be great if we could avoid using specific-markdown(like hyperlinks), wherever possible.

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+**Make sure to tell your friends about it, so that they also learn something new!**
+
+## YouTube Shorts
+I have another YouTube channel for short clips, please **take a moment subscribing it: [youtube.com/eddiejaoudetv](https://youtube.com/eddiejaoudetv?sub_confirmation=1)**
```
```suggestion
I have another YouTube channel for short clips, please **take a moment subscribing it: https://youtube.com/eddiejaoudetv?sub_confirmation=1**
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+I have another YouTube channel for short clips, please **take a moment subscribing it: [youtube.com/eddiejaoudetv](https://youtube.com/eddiejaoudetv?sub_confirmation=1)**
+
+## Join Our Community
+Join our inclusive open source community EddieHub now for **free** at [eddiehub.org](https://www.eddiehub.org)
```
```suggestion
Join our inclusive open source community EddieHub now for **free** at https://www.eddiehub.org
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+ 
+---
+
+Make sure to checkout my website and sign up to our community **newsletter**: [eddiejaoude.io](https://www.eddiejaoude.io) for my live stream + video schedule and community calls
```
```suggestion
Make sure to checkout my website and sign up to our community **newsletter**: https://www.eddiejaoude.io for my live stream + video schedule and community calls
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+
+Join Our **Community GitHub Organization**:
+ 
+- Go to [EddieHubCommunity/support/issues/new/choose](https://github.com/EddieHubCommunity/support/issues/new/choose) 
```
```suggestion
- Go to https://github.com/EddieHubCommunity/support/issues/new/choose 
```

Vyvy-vi replied:
These unordered lists don't render properly when copied and pasted. Maybe we can edit and make it look like-
```suggestion
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**
```
this renders- 
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
\- Choose The Issue Type Of **Invite Me To The Organization**

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+---
+
+### GitHub Sponsors
+- Stephen Mount: [github.com/stemount](https://github.com/stemount)
```
```suggestion
- Stephen Mount: https://github.com/stemount
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+
+### GitHub Sponsors
+- Stephen Mount: [github.com/stemount](https://github.com/stemount)
+- Nicholas Carrigan: [github.com/nhcarrigan](https://github.com/nhcarrigan)
```
```suggestion
- Nicholas Carrigan: https://github.com/nhcarrigan
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+### GitHub Sponsors
+- Stephen Mount: [github.com/stemount](https://github.com/stemount)
+- Nicholas Carrigan: [github.com/nhcarrigan](https://github.com/nhcarrigan)
+- Santosh Yadav: [github.com/santoshyadavdev](https://github.com/santoshyadavdev)
```
```suggestion
- Santosh Yadav: https://github.com/santoshyadavdev
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- Stephen Mount: [github.com/stemount](https://github.com/stemount)
+- Nicholas Carrigan: [github.com/nhcarrigan](https://github.com/nhcarrigan)
+- Santosh Yadav: [github.com/santoshyadavdev](https://github.com/santoshyadavdev)
+- Andrew Cunliffe: [github.com/andrew-cunliffe](https://github.com/andrew-cunliffe)
```
```suggestion
- Andrew Cunliffe: https://github.com/andrew-cunliffe
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- Nicholas Carrigan: [github.com/nhcarrigan](https://github.com/nhcarrigan)
+- Santosh Yadav: [github.com/santoshyadavdev](https://github.com/santoshyadavdev)
+- Andrew Cunliffe: [github.com/andrew-cunliffe](https://github.com/andrew-cunliffe)
+- Alexander Stoichkov: [github.com/SashoStoichkov](https://github.com/SashoStoichkov)
```
```suggestion
- Alexander Stoichkov: https://github.com/SashoStoichkov
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- Santosh Yadav: [github.com/santoshyadavdev](https://github.com/santoshyadavdev)
+- Andrew Cunliffe: [github.com/andrew-cunliffe](https://github.com/andrew-cunliffe)
+- Alexander Stoichkov: [github.com/SashoStoichkov](https://github.com/SashoStoichkov)
+- Allan Regush: [github.com/AllanRegush](https://github.com/AllanRegush)
```
```suggestion
- Allan Regush: https://github.com/AllanRegush
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+## Social Media
+Follow on socials for behind the scenes footage and more.
+
+- Website: [eddiejaoude.io](https://www.eddiejaoude.io) 
```
```suggestion
- Website: https://www.eddiejaoude.io 
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+Follow on socials for behind the scenes footage and more.
+
+- Website: [eddiejaoude.io](https://www.eddiejaoude.io) 
+- GitHub: [github.com/eddiejoude](https://github.com/eddiejaoude)
```
```suggestion
- GitHub: https://github.com/eddiejaoude
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+
+- Website: [eddiejaoude.io](https://www.eddiejaoude.io) 
+- GitHub: [github.com/eddiejoude](https://github.com/eddiejaoude)
+- YouTube: [youtube.com/eddiejoude](https://youtube.com/eddiejaoude)
```
```suggestion
- YouTube: https://youtube.com/eddiejaoude
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- Website: [eddiejaoude.io](https://www.eddiejaoude.io) 
+- GitHub: [github.com/eddiejoude](https://github.com/eddiejaoude)
+- YouTube: [youtube.com/eddiejoude](https://youtube.com/eddiejaoude)
+- Twitter: [twitter.com/eddiejaoude](https://twitter.com/eddiejaoude)
```
```suggestion
- Twitter: https://twitter.com/eddiejaoude
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- GitHub: [github.com/eddiejoude](https://github.com/eddiejaoude)
+- YouTube: [youtube.com/eddiejoude](https://youtube.com/eddiejaoude)
+- Twitter: [twitter.com/eddiejaoude](https://twitter.com/eddiejaoude)
+- Instagram: [instagram.com/eddiejaoude](https://instagram.com/eddiejaoude)
```
```suggestion
- Instagram: https://instagram.com/eddiejaoude
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- YouTube: [youtube.com/eddiejoude](https://youtube.com/eddiejaoude)
+- Twitter: [twitter.com/eddiejaoude](https://twitter.com/eddiejaoude)
+- Instagram: [instagram.com/eddiejaoude](https://instagram.com/eddiejaoude)
+- LinkedIn: [linkedin.com/in/eddiejaoude](https://linkedin.com/in/eddiejaoude)
```
```suggestion
- LinkedIn: https://linkedin.com/in/eddiejaoude
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- Twitter: [twitter.com/eddiejaoude](https://twitter.com/eddiejaoude)
+- Instagram: [instagram.com/eddiejaoude](https://instagram.com/eddiejaoude)
+- LinkedIn: [linkedin.com/in/eddiejaoude](https://linkedin.com/in/eddiejaoude)
+- GitHub Community Org: [github.com/EddieHubCommunity](https://github.com/EddieHubCommunity)
```
```suggestion
- GitHub Community Org: https://github.com/EddieHubCommunity
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,46 @@
+- GitHub Community Org: [github.com/EddieHubCommunity](https://github.com/EddieHubCommunity)
+
+## Further
+Join Our Conversation At Discord: [discord.eddiehub.org](https://discord.eddiehub.org/)
```
```suggestion
Join Our Conversation At Discord: https://discord.eddiehub.org/
```

Vyvy-vi reviewed (changes requested):
I looked at the PR and left a few comments for changes :+1:

krishdevdb commented:
Commited all yours changes @Vyvy-vi I had a question, shall I rename it to a txt extension, for easy copying

Vyvy-vi commented:
> Commited all yours changes @Vyvy-vi I had a question, shall I rename it to a txt extension, for easy copying

I think either way works, but keeping it markdown might be more consistent

Vyvy-vi reviewed (dismissed):
LGTM!

krishdevdb commented:
> markdown might be more consistent
We can open an issue and then maybe later a pr which does the change?

deephunt3r commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+- GitHub Community Org: https://github.com/EddieHubCommunity
+
+## Further
+Join Our Conversation At Discord: https://discord.eddiehub.org/
```
Hey! The discord link isn't working, I have checked with the working link it contains **http** I wonder why it happens
have a look at it once [http://discord.eddiehub.org/](http://discord.eddiehub.org/)

Vyvy-vi replied:
oh wupps slipped past me - http:///discord.eddiehub.org/ would be the correct link

deephunt3r replied:
@Vyvy-vi may I know why it is not working for https 🤔

schmelto replied:
> @Vyvy-vi may I know why it is not working for https 🤔 

Why?

Vyvy-vi replied:
The discord link sub-domain is a URL-forward, instead of a normal CNAME or A level record(since discord's invite is case sensitive).
And thus, I think it doesn't get the SSL cert. from the root domain(if Eddie's provider supports that thing)
But, if Eddie's domain provider doesn't support inheriting SSL certs to sub-domains, then this just ends up as http.

There might be a way to make this work with https tho, with something like cloudfare acting as an intermediary

nhcarrigan replied:
It's a DNS level redirect, which doesn't support HTTPS. You can't use a CNAME or A record because those don't accept URL paths. 

It would be possible, however, to spin up a small web server to handle the redirect and maintain HTTPS.

deephunt3r reviewed (approved):
LGTM 🚀

schmelto commented:
Ups sorry clicked the wrong button 🙈

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+
+Join Our **Community GitHub Organization**:
+ 
+- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
```
I think we would want these to appear as bullet points on youtube too. (normal bullets in markdown get copied as normal text, but we could ignore this line to get a copy-able list)
```suggestion
\- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+Join Our **Community GitHub Organization**:
+ 
+- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+- Choose The Issue Type Of **Invite Me To The Organization**
```
```suggestion
\- Choose The Issue Type Of **Invite Me To The Organization**
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+ 
+- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Choose The Issue Type Of **Invite Me To The Organization**
```
```suggestion
\- Choose The Issue Type Of **Invite Me To The Organization**
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+- Go to https://github.com/EddieHubCommunity/support/issues/new/choose
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Open The Issue
```
```suggestion
\- Open The Issue
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,37 +1,47 @@
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Choose The Issue Type Of **Invite Me To The Organization**
+- Open The Issue
+- You get an invitation to the community by our GitHub Action 🎉
```
```suggestion
\- You get an invitation to the community by our GitHub Action 🎉
```

Vyvy-vi reviewed (commented):
Left a few comments realted to rendering bullet points in the youtube discussion.
Thoughts? 

*this can be done in a later PR too :)

schmelto reviewed (dismissed):
Looking good for me but I think it's up to @eddiejaoude to merge it cause I think this is the description for his yt videos

Vyvy-vi reviewed (dismissed):
LGTM, but I guess it's best that Eddie reviews this before a merger, since the content changed a bit in tone

vinzvinci reviewed (approved):
Looks good to me!

kaiwalyakoparkar reviewed (approved):
LGTM :rocket:

eddiejaoude commented:
Thank you everyone 👍 

Unfortunately, YouTube description does not allow for markdown, so I used `""` or capitals instead. Please remove markdown

krishdevdb commented:
> Thank you everyone 👍
> 
> Unfortunately, YouTube description does not allow for markdown, so I used `""` or capitals instead. Please remove markdown

Asteriks can do the same work as per me, asteriks are something I see more often for that purpose of highlighing

Vyvy-vi commented:
I'm not sure if asterisk are friendly for screen readers and such 🤔

krishdevdb commented:
Resolved all conflicts 😌! Can you review again folks?

krishdevdb commented:
> Thank you everyone 👍
> 
> Unfortunately, YouTube description does not allow for markdown, so I used `""` or capitals instead. Please remove markdown

Done, remove the markdown

krishdevdb commented:
> I'm not sure if asterisks are friendly for screen readers and such 🤔

Thanks for pointing out, I removed the asterisks and replaced them with capitals

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,49 +1,57 @@
-https://www.youtube.com/eddiejaoude?sub_confirmation=1
-
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+Some of these might be affiliate links, no extra cost you to, but I get a little $ if you sign up, plus you might get a discount too
```
```suggestion
Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,49 +1,57 @@
 Stephen Mount http://github.com/stemount
 Nicholas Carrigan https://github.com/nhcarrigan
-***
+Santosh Yadav https://github.com/santoshyadavdev
```
while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -
```suggestion
\- Santosh Yadav https://github.com/santoshyadavdev
```

Vyvy-vi replied:
somewhat like this-
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount
```
\- Santosh Yadav https://github.com/santoshyadavdev
\- Stephen Mount http://github.com/stemount

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,49 +1,57 @@
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
```
while copy pasting, all these bullets get added to the same line.
A solution might be to split them into new lines -

```suggestion
\- NoSQL serverless database with DataStax, you will get $25 FREE credit every month: http://links.eddiejaoude.io/t/bBM0STvCi2a4
```

This would prevent this to get weirdly formatted while copy-pasting from the markdown file:

![Screenshot 2021-06-25 at 1 26 12 PM](https://user-images.githubusercontent.com/62864373/123390917-1309af80-d5b9-11eb-979a-d413a92be8b5.png)

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,49 +1,75 @@
+1. Go to https://github.com/EddieHubCommunity/support/issues/new/choose
 
-## SERVICES I USE AND RECOMMEND
+2. Choose The Issue Type Of Invite Me To The Organization
```
points 2 and 3 are duplicates

krishdevdb commented:
I made a lot of conflicts while fixing merge conflicts 🤦🏻

Akshu-on-github commented on youtube/generic.md:
```diff
@@ -41,15 +41,11 @@ Allan Regush https://github.com/AllanRegush
 
 ## Service I Use and Recommend
 
-Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
```
This sounds like a disclaimer - are we sure we can leave this out?

eddiejaoude replied:
I think most people write something above their affiliate links, to avoid any confusion

krishdevdb commented:
@Akshu-on-github I messed up with the commit message but fixed the issue. Can you review again?

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
 
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY ALSO LEARN SOMETHING NEW!
```
```suggestion
MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY CAN ALSO LEARN SOMETHING NEW!
```
I think maybe re-phrasing like that might sound better?

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
 
-## LEARN MORE
+## YouTube Shorts
+I have another YouTube channel for short clips, please TAKE A MOMENT SUBSCRIBING IT: https://youtube.com/eddiejaoudetv?sub_confirmation=1
```
```suggestion
I have another YouTube channel for short clips, please take a moment subscribing to it: https://youtube.com/eddiejaoudetv?sub_confirmation=1
```
I think the capitals might not look very natural. (Non-violent-communication in phrasing like these might look better). Thoughts, @eddiejaoude ?

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+## Community Sponsors
 
-A BIG thank you to my GitHub Sponsors:
-***
```
I'm not sure, but I think those Asterisks were there since Stephen and Nicholas are Eddie's biggest sponsors, or something like that. _Not sure_

krishdevdb replied:
I also kept the difference but I was asked by ig you or @kaiwalyakoparkar to remove the diff

krishdevdb replied:
ig maybe we can ask @eddiejaoude for to keep the diff/not keep the diff

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+
+---
+ 
+## Find me on social media
```
I think I'm nitpicking here, but Eddie usually refers to them as `socials` 😅:
```suggestion
## Find me on socials
```

Vyvy-vi commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## Further
+Join Our Conversation At Discord: http://discord.eddiehub.org/
```
```suggestion
Join the Conversation at Discord: http://discord.eddiehub.org/
```
I think this re-phrasing might be grammatically neater. Thoughts?

Vyvy-vi reviewed (commented):
I added a few comments. These aren't "requests for changes", but rather some questions/discussions about things. Many of these comments are nit-picky, but I think some of them might hold value, if discussed/considered

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
-...
+## YouTube Channel
+Don’t forget to support my channel by subscribing it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```
Hmm, shouldn't this be `'` instead of `’`?
```suggestion
Don't forget to support my channel by subscribing to it! It’s free: https://youtube.com/eddiejaoude?sub_confirmation=1
```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+
+---
+
+## Service I Use and Recommend
```
```suggestion
## Services I use and recommend
```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
-- Allan Regush https://github.com/AllanRegush
+Santosh Yadav https://github.com/santoshyadavdev
+
+Andrew Cunliffe https://github.com/andrew-cunliffe
+
+Alexander Stoichkov https://github.com/SashoStoichkov
+
+Allan Regush https://github.com/AllanRegush
```
```suggestion
[Andrew Cunliffe](https://github.com/andrew-cunliffe)\
[Alexander Stoichkov](https://github.com/SashoStoichkov)\
[Allan Regush](https://github.com/AllanRegush)
```

That would make it look like

[Andrew Cunliffe](https://github.com/andrew-cunliffe)\
[Alexander Stoichkov](https://github.com/SashoStoichkov)\
[Allan Regush](https://github.com/AllanRegush)

krishdevdb replied:
Markdown is not supported in yt vid descriptions, sorry

Panquesito7 replied:
Oh, I didn't know this was for YouTube videos 😅 Sorry.

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+
+## Service I Use and Recommend
+
+Some of these might be affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount too
```
```suggestion
Some of these might be an affiliate links, no extra cost to you, but I get a little $ if you sign up, plus you might get a discount, too.
```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+ 
+## Find me on socials
+
+Website: https://www.eddiejaoude.io 
+
+GitHub: https://github.com/eddiejaoude
+
+YouTube: https://youtube.com/eddiejaoude
+
+Twitter: https://twitter.com/eddiejaoude
```
```suggestion
Website: https://www.eddiejaoude.io\
GitHub: https://github.com/eddiejaoude\
YouTube: https://youtube.com/eddiejaoude\
Twitter: https://twitter.com/eddiejaoude\
```

krishdevdb replied:
Tried but unacceptedly, he whole thing appears on one line

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+YouTube: https://youtube.com/eddiejaoude
+
+Twitter: https://twitter.com/eddiejaoude
 
```
```suggestion
```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
-http://linkedin.com/in/eddiejaoude
-https://discord.com/invite/jZQs6Wu
-http://github.com/EddieHubCommunity
+## Further
```
Maybe `miscellaneous` would be better instead?
```suggestion
## Miscellaneous
```

krishdevdb replied:
Futher seems a little more friendly to me. Miscellaneous will be a little to much formal context to the actual text inside

Panquesito7 replied:
What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

krishdevdb replied:
> What about `more information`? Or `extras` or something like that? I feel `further` doesn't fit well there.

These words too aren't a fit here. But maybe **Meet our Community** will work?

Panquesito7 replied:
That sounds great! 😄

Vyvy-vi replied:
Essentially, the Heading isn't really necessary
You could state it without a heading

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,71 @@
+LinkedIn: https://linkedin.com/in/eddiejaoude
 
-Follow on other socials for behind the scenes footage, join discord to continue the conversation...
+GitHub Community Org: https://github.com/EddieHubCommunity
```
```suggestion
GitHub Community Organization link: https://github.com/EddieHubCommunity
```

Panquesito7 reviewed (changes requested):
Just minor reviews. Most of them are just for the text to look better. 🙂

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,70 @@
+MAKE SURE TO TELL YOUR FRIENDS ABOUT IT, SO THAT THEY CAN ALSO LEARN SOMETHING NEW!
 
-## LEARN MORE
+## YouTube Shorts
```
```suggestion
## YouTube Shorts

```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,70 @@
+Make sure to checkout my website and sign up to our community NEWSLETTER: https://www.eddiejaoude.io for my live stream + video schedule and community calls
 
-Also join our community GitHub organisation by creating an issue with the type "invite me to the organisation" :)
+## Join Our Community
```
```suggestion
## Join Our Community

```

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,70 @@
-...
+## YouTube Channel
```
```suggestion
# YouTube Channel
```

Panquesito7 reviewed (commented):
Just some minor backend suggestions.

Panquesito7 commented on youtube/generic.md:
```diff
@@ -1,50 +1,72 @@
-...
+# YouTube Channel
```
```suggestion
# YouTube Channel

```

Panquesito7 reviewed (approved):
Looks good. Thanks! 😄

adityaraute reviewed (commented):
As an advice for future PRs, keep your changes short. Generate more PRs if necessary but bulky ones like these take a lot of time.

krishdevdb commented:
> As an advice for future PRs, keep your changes short. Generate more PRs if necessary but bulky ones like these take a lot of time.

The Pull Request was just one commit but changes coming from other folks in pull #1615 and now in this pull made this pull a little larger. The pull was only meant to fix #1582

Akshu-on-github reviewed (approved):
I can see a few points for improvement, but I think it would be best to merge it now - the original intention of the PR is satisfied.

Vyvy-vi commented:
Since there's been a large amount of discussion here, and Eddie didn't reject any changes(and this has been open for a while), I think it's good to go
